src/views/Explore.vue

```diff
@@ -3,7 +3,7 @@
     <div class="mb-4 mx-auto">
       <Container class="d-flex flex-items-center">
         <div class="flex-auto text-left">
-          <UiButton class="pl-3 col-12 col-lg-4">
+          <UiButton class="pl-3 col-12 col-lg-6">
```

Comment:
The size of the search input change depending on the page, this should not happen, it should always be the same size regardless of the page we are in.

![image](https://user-images.githubusercontent.com/16245250/111898587-9a470480-8a59-11eb-8788-c21b0faf160f.png)
![image](https://user-images.githubusercontent.com/16245250/111898589-a337d600-8a59-11eb-80e0-e268297d1065.png)